openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -247,7 +250,9 @@ def check_permissions(self, instance: dict[str, Any]) -> None:
                 lock_result=False,
             ).get("locked_from_inside", False)
 
-        # Ordered by supposed velocity advantages. Changing order can only effect the sequence of detected errors for tests
+        # Ordered by supposed speed advantages. Changing order can only effect the sequence of detected errors for tests
+        if self._meeting_admin_can_manage_non_admin():
```

Comment:
This should not skip the field group checks, but rather be part of them.
You should look into the action description to see what field groups need to be changed and then only change those.
You should especially not be skipping the checks for that have nothing to do with what our issue management people usually call `personal information`, f.E. groups `E`, `G`, and `H`.

Also think of maybe moving this functionality to the `UserScopeMixin` and using it for multiple classes.